### ISSUE TITLE
can not download kogito-tooling/build-env from npm repository

### ISSUE THREAD

**Issue by fangjiangz** [apache/incubator-kie-tools#727]
I am installing project kie-editors-standalone，it failed， because that kogito-tooling/build-env  cannot be download. I searched it on npmjs, it cannot be found. could you kindly upload kogito-tooling/build-env package to npmjs?

**Comment by tiagobento** [comment]
Hi @fangjiangz! `@kogito-tooling/build-env` is always a "devDependency", so it shouldn't be being transitively downloaded when you install `@kogito-tooling/kie-editors-standalone`. Can you please provide here the command you used and the output you got?

**Comment by fangjiangz** [comment]
thanks @tiagobento 。I followed the blow guide in readme. 
#### Standalone Editors

1. After you've successfully built the project following the instructions above, go to `packages/kie-editors-standalone`.
2. Open a terminal and run `yarn start`. This will start a `webpack serve` instance with the Standalone Editors test page.
3. From now on you can use the development version of the Standalone DMN Editor by accessing `https://localhost:9001/resources/dmn` and the Standalone BPMN Editor by accessing `https://localhost:9001/resources/bpmn`.
###########

but when I run '`yarn start` , i got the below error message. I want to know what steps I am missing.

###############
FangdeMacBook-Pro:kie-editors-standalone fangjian$ yarn start
yarn run v1.22.17
$ webpack serve --host 0.0.0.0 --config webpack.package-resources.config.js
Webpack :: ts-loader :: transpileOnly: false
Webpack :: minimize: true
Webpack :: sourceMaps: false
Webpack :: mode: production
Webpack :: live: undefined
ℹ ｢wds｣: Project is running at http://0.0.0.0:9006/
ℹ ｢wds｣: webpack output is served from /
ℹ ｢wds｣: Content not from webpack is served from /Users/fangjian/work/kogito-tooling-main/packages/kie-editors-standalone/dist
✖ ｢wdm｣: assets by status 405 KiB [cached] 28 assets
hidden modules 439 KiB (javascript) 1.74 KiB (runtime) 78 modules

ERROR in ./src/bpmn/index.ts 15:31-103
Module not found: Error: Can't resolve '../../dist/resources/bpmn/bpmnEnvelopeIndex.html' in '/Users/fangjian/work/kogito-tooling-main/packages/kie-editors-standalone/src/bpmn'
resolve '../../dist/resources/bpmn/bpmnEnvelopeIndex.html' in '/Users/fangjian/work/kogito-tooling-main/packages/kie-editors-standalone/src/bpmn'
  using description file: /Users/fangjian/work/kogito-tooling-main/packages/kie-editors-standalone/package.json (relative path: ./src/bpmn)
    Field 'browser' doesn't contain a valid alias configuration
    using description file: /Users/fangjian/work/kogito-tooling-main/packages/kie-editors-standalone/package.json (relative path: ./dist/resources/bpmn/bpmnEnvelopeIndex.html)
      no extension
        Field 'browser' doesn't contain a valid alias configuration
        /Users/fangjian/work/kogito-tooling-main/packages/kie-editors-standalone/dist/resources/bpmn/bpmnEnvelopeIndex.html doesn't exist
      .tsx
        Field 'browser' doesn't contain a valid alias configuration
        /Users/fangjian/work/kogito-tooling-main/packages/kie-editors-standalone/dist/resources/bpmn/bpmnEnvelopeIndex.html.tsx doesn't exist
      .ts
        Field 'browser' doesn't contain a valid alias configuration
        /Users/fangjian/work/kogito-tooling-main/packages/kie-editors-standalone/dist/resources/bpmn/bpmnEnvelopeIndex.html.ts doesn't exist
      .js
        Field 'browser' doesn't contain a valid alias configuration
        /Users/fangjian/work/kogito-tooling-main/packages/kie-editors-standalone/dist/resources/bpmn/bpmnEnvelopeIndex.html.js doesn't exist
      .jsx
        Field 'browser' doesn't contain a valid alias configuration
        /Users/fangjian/work/kogito-tooling-main/packages/kie-editors-standalone/dist/resources/bpmn/bpmnEnvelopeIndex.html.jsx doesn't exist

**Comment by tiagobento** [comment]
@fangjiangz Thanks for the details. I still don't understand whether you're trying to consume the `@kogito-tooling/kie-editors-standalone` package or develop it. Can you please clarify?

**Comment by ederign** [comment]
Closing due to inactivity. Feel free to reopen if necessary.

**Comment by Happy-Child** [comment]
![image](https://github.com/apache/incubator-kie-tools/assets/40471724/e8cf91a2-a38f-47f2-8601-206c27e7510e)
![image](https://github.com/apache/incubator-kie-tools/assets/40471724/ec11ec7d-6f4c-454a-aaa2-0c8d6346f817)


HelloI have the same situation, here are the details:
I want to run the bpmn editor project locally.
I followed the instructions https://github.com/apache/incubator-kie-tools/blob/main/README.md#build-from-source

Start - pnpm bootstrap
Then I proceed https://github.com/apache/incubator-kie-tools/blob/main/README.md#build-from-source - I go to kie-editors-standalone and run pnpm start.
I get a lot of TypeScript errors and errors about missing files, I've attached a screenshot.

I've also attached a screenshot of the successful completion of pnpm bootstrap.
I didn't install some utilities, java/helm/make, but I don't think the problem lies in that.

I am trying to run the project locally for development.
Thank you.
